### DISCUSSION TITLE
Elevenlabs Integration has stopped working

### DISCUSSION THREAD

**Discussion by michaelwlt** [open-webui/open-webui#6551] [Issues]
Hey, I am not sure whether this problem is on me or something has changed in the recent releases. 
I've used the Elevenlabs API for the TTS in the past without any issues. After not using the feature for a while I wanted to use it again. However, all I am getting now are 401 client errors. 

## What I already have checked
- I've regenerated my API key and made sure all necessary permissions are given
- I've made sure I have not exceeded my monthly quota
- I've tried to use a different voice ID and voice model

## Additional information
- I am running my open webui instance on render.com
- I am on the free tier of elevenlabs

Any help is appreciated
Here are my logs:
![image](https://github.com/user-attachments/assets/fbd984e4-13d1-4bff-9e56-dbea30e95a83)

**Comment by darkBuddha** [comment]
Nofix?
